cypress/support/step-definitions/flat-table-steps.js

```diff
@@ -98,3 +100,91 @@ Then('I focus {int} row and focused row element has golden border on focus', (in
+      .and('be.visible');
+    flatTableCell(positionOfElement('first')).should('have.text', valueOne)
+      .and('be.visible');
+    flatTableCell(positionOfElement('third')).should('have.text', valueTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fifth')).should('have.text', valueThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('seventh')).should('have.text', valueFour)
+      .and('be.visible');
+  } else if (position === 'first' && sortOrder === 'asc') {
+    icon().should('have.attr', 'data-element', 'sort_up')
+      .and('be.visible');
+    flatTableCell(positionOfElement('first')).should('have.text', valueFour)
+      .and('be.visible');
+    flatTableCell(positionOfElement('third')).should('have.text', valueThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fifth')).should('have.text', valueTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('seventh')).should('have.text', valueOne)
+      .and('be.visible');
+  } else if (position === 'second' && sortOrder === 'desc') {
+    icon().should('have.attr', 'data-element', 'sort_down')
+      .and('be.visible');
+    flatTableCell(positionOfElement('second')).should('have.text', totalOne)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fourth')).should('have.text', totalTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('sixth')).should('have.text', totalThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('eighth')).should('have.text', totalFour)
+      .and('be.visible');
+  } else {
+    icon().should('have.attr', 'data-element', 'sort_up')
+      .and('be.visible');
+    flatTableCell(positionOfElement('second')).should('have.text', totalFour)
+      .and('be.visible');
+    flatTableCell(positionOfElement('fourth')).should('have.text', totalThree)
+      .and('be.visible');
+    flatTableCell(positionOfElement('sixth')).should('have.text', totalTwo)
+      .and('be.visible');
+    flatTableCell(positionOfElement('eighth')).should('have.text', totalOne)
+      .and('be.visible');
+  }
+});
+
+Then('Flat table header has {string} color', (colorTheme) => {
+  flatTableHeaderCells().each(($el) => {
+    cy.wrap($el).should('have.css', 'background-color', colorTheme);
+  });
+});
+
+Then('{string} header has focus', (position) => {
+  flatTableHeaderCells().find('div').eq(positionOfElement(position)).should('have.css', 'outline-color', 'rgb(255, 181, 0)');
+});
+
+Then('I focus {string} header cell', (position) => {
+  flatTableHeaderCells().find('div').eq(positionOfElement(position)).focus();
+});
+
+Then('I press {string} on {string} header {int} time(s)', (key, position, count) => {
+  for (let i = 0; i < count; i++) {
+    if (key === 'Enter') {
+      flatTableHeaderCells().find('div').eq(positionOfElement(position)).focus()
+        .trigger('keydown', { keyCode: 13, which: 13, force: true });
+    } else {
```

Comment:
add please `else if`
```suggestion
    } else if (key === 'Space' ) {
```

Reply:
fixed